resources/views/includes/check-in-modal.blade.php

```diff
@@ -0,0 +1,106 @@
+                        });
+                    </script>
+
+                    <div class="mt-2">
+                        @if (auth()->user()->socialProfile != null)
+                            @if (auth()->user()->socialProfile->twitter_id != null)
+                                <div class="btn-group">
+                                    <input type="checkbox" class="btn-check" id="tweet_check" autocomplete="off"
+                                           name="tweet_check"/>
+                                    <label class="btn btn-sm btn-outline-twitter" for="tweet_check">
+                                        <i class="fab fa-twitter"></i>
+                                        <span
+                                            class="visually-hidden-focusable">{{ __('stationboard.check-tweet') }}</span>
+                                    </label>
+                                </div>
+                            @endif
+
+                            @if (auth()->user()->socialProfile->mastodon_id != null)
+                                <div class="btn-group">
+                                    <input type="checkbox" class="btn-check" id="toot_check" autocomplete="off"
+                                           name="toot_check"/>
+                                    <label class="btn btn-sm btn-outline-mastodon" for="toot_check">
+                                        <i class="fab fa-mastodon"></i>
+                                        <span class="visually-hidden-focusable">
+                                                {{ __('stationboard.check-toot') }}
+                                            </span>
+                                    </label>
+                                </div>
+                                @if (\App\Http\Controllers\Backend\Social\MastodonController::getLastSavedPostIdFromUserStatuses(auth()->user()))
+                                    <div class="btn-group">
+                                        <input type="checkbox" class="btn-check" id="chainPost_check" autocomplete="off"
+                                               name="chainPost_check"/>
+                                        <label class="btn btn-sm btn-outline-mastodon" for="chainPost_check">
+                                            <i class="fa-solid fa-list-ol"></i>
+                                            <span class="visually-hidden-focusable">
+                                                    {{ __('stationboard.check-chainPost') }}
+                                                </span>
+                                        </label>
+                                    </div>
+                                @endif
+                            @endif
+                        @endif
+                        @include('includes.business-dropdown')
+                        @include('includes.visibility-dropdown')
+                    </div>
+
```

Comment:
> I thought whether there ist a better way to so this, but i don't know PHP good enough to know how to do this

You can just set `$events` it in the view itself:

```suggestion

@php($events = \App\Http\Controllers\Backend\EventController::activeEvents())
```

Reply:
Oh so that's how this works thx